### DISCUSSION TITLE
calling save() on Livewire instance gives maximum execution 60 error instead of throwing errors/exception

### DISCUSSION THREAD

**Discussion by charles-prof** [livewire/livewire#4990] [1. Help]
To persist a model $this->model->save() is used on the Livewire save() method.

I just changed it to $this->save() to check can livewire can automatically persist the model without calling ($this->model)

I used telescope to debug but couldn't identify what error and how it behaves.

I use php on windows (not apache), is there any server log for php local installation ?

**Comment by joshhanley** [comment]
@charles-prof you need to call `$this->model->save()` otherwise the model won't be persisted to the database.

By calling `$this->save()`, it is just calling itself again, which is creating an infinite loop, hence why you are getting a execution timeout.

Hope this helps!
